deepchecks/tabular/checks/distribution/train_test_label_drift.py

```diff
@@ -49,6 +55,8 @@ class TrainTestLabelDrift(TrainTestCheck):
         - 'train_largest': Show the largest train categories.
         - 'test_largest': Show the largest test categories.
         - 'largest_difference': Show the largest difference between categories.
+    categorical_drift_method: str, default: "Cramer"
+        Cramer for Cramer's V, PSI for Population Stability Index (PSI).
```

Comment:
Should first describe what this does. "decides which method to use on categorical variables. Possible values are: "